examples/derived.ts

```diff
@@ -0,0 +1,16 @@
+import { derive } from "../core/unsafe/mod.ts";
+import * as Z from "../mod.ts";
+
+const coinToss = Z.call(0, () => {
+  return Math.random() > .5;
+});
```

Comment:
Utility suggestion
```
export function nullaryCall<R>(fn: CallLogic<undefined, R>) {
  return new Call(undefined, fn);
}
```

```
const coinToss = Z.nullaryCall(() => {
  return Math.random() > .5;
});
```

Reply:
or update 
```
Z.call(0, () => { ... })
```
to
```
Z.call(undefined, () => { ... })
Z.call(void 0, () => { ... })
```

Reply:
`0` takes less space. Not sure we should introduce a utility for nullary calls... imo, overcomplicates it more than just passing `0` or `undefined` (I personally like `0`).